android/src/main/java/com/reactlibrary/RNSpotifyRemoteAppModule.java

```diff
@@ -98,8 +99,14 @@ public void onEvent(String eventName, Object... args) {
     private void sendEvent(String eventMame, Object data) {
         RNEventEmitter.emitEvent(this.reactContext, this, eventMame, data);
     }
-
+    
     private void handleOnConnect() {
+        mSpotifyAppRemote.getPlayerApi()
```

Comment:
At some point it would probably be good to subscribe to these events lazily when people subscribe from the javascript side.  The ios implementation already does this, however I think I got lazy at the time of finishing up the android side.

Reply:
Good suggestion, perhaps open a new issue to keep track of this one?

Reply:
Created #122 for tracking this one.